packages/models/src/identity/index.ts

```diff
@@ -94,6 +96,27 @@ export class IdentityModel extends BaseModel {
     return newProof;
   }
 
+  public async getIdentitiesByDisplayName(
+    dn: string
+  ): Promise<IdentityRecord[]> {
+    const KeyConditionExpression: DocumentClient.KeyExpression =
+      'pk = :displayName';
+    const ExpressionAttributeValues: DocumentClient.ExpressionAttributeValueMap = {
+      ':displayName': `${dn}`,
+    };
+
+    const params = {
+      TableName: this.table,
+      KeyConditionExpression,
+      ExpressionAttributeValues,
+      IndexName: 'gs1',
```

Comment:
is this correct index? I mean isn't that one used with gs1pk and gs1sk?

Reply:
Good catch. Fixed in [8518135](https://github.com/FleekHQ/space-services/pull/36/commits/8518135f7d655c08c7b280567e1d37db344889d4).